modules/network-observability-create-network-policy.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/configuring-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-network-policy_{context}"]
+= Secure Network Observability with a network policy using the web console
+You may need to create a network policy to secure ingress traffic to the 'netobserv' namespace. Using the web console, you can create a network policy 
+
+.Procedure
+. Navigate to *Networking* -> *NetworkPolicies*.
+. Select the `netobserv` project from the *Project* dropdown menu.
+. Name the policy. For this example, the policy name is `allow-ingress`.
+. Click the *Add ingress rule* button twice to create two ingress rules, and make the following specifications in the form:
+.. Make the following specifications for the first *Ingress rule*:
+... From the *Add allowed source* dropdown menu, select *Allow pods from inside the cluster*. 
+... Click *+ Add namespace selector*. 
+... Add the label, `kubernetes.io/metadata.name`, and the selector, `openshift-console`. 
+.. Make the following specifications for the second *Ingress rule*:
+... From the *Add allowed source* dropdown menu, select *Allow pods from inside the cluster*.
+... Click *+ Add namespace selector*. 
+... Add the label, `kubernetes.io/metadata.name`, and the selector, `openshift-monitoring`.
+
+.Optional
+You can edit the network policy using the YAML editor in the web console. The YAML file should be like the following sample:
```

Comment:
@stleerh for the YAML piece, I know in your video, you created the `allow-ingress` policy, then you edited it in the YAML. In my example, I went ahead and put the `openshift-monitoring` in the initial setup and then as an Optional step, say you can edit the policy using the YAML editor.